src/main/java/core/basesyntax/service/strategy/OperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.dto.FruitTransactionDto;
+import java.util.HashMap;
+
+public interface OperationHandler {
+    HashMap<String, Integer> apply(FruitTransactionDto dto);

```

Comment:
handler -> handle, we also do not need to return anything, just update storage in it
```suggestion
public interface OperationHandler {
    void handle(FruitTransactionDto dto);
```